config/patches/datadog-metro/libpcap-static-link.patch

```diff
@@ -0,0 +1,18 @@
+diff -ur a/pcap/pcap.go b/pcap/pcap.go
+--- a/pcap/pcap.go	2015-12-30 02:02:44.710877403 +0000
++++ b/pcap/pcap.go	2015-12-30 02:11:42.769664808 +0000
+@@ -8,11 +8,12 @@
+ package pcap
+ 
+ /*
+-#cgo linux LDFLAGS: -lpcap
++#cgo linux CFLAGS: -I/opt/datadog-agent/embedded/include
++#cgo linux LDFLAGS: /opt/datadog-agent/embedded/lib/libpcap.a
+ #cgo dragonfly LDFLAGS: -lpcap
+ #cgo freebsd LDFLAGS: -lpcap
+ #cgo openbsd LDFLAGS: -lpcap
+-#cgo darwin LDFLAGS: -lpcap
++#cgo darwin LDFLAGS: /usr//local/Cellar/libpcap/1.7.4/lib/libpcap.a
```

Comment:
let's add a `TODO OSX support` comment in `config/software/datadog-metro.rb` mentioning this so that we don't forget :)